src/about.njk

```diff
@@ -101,7 +101,7 @@
         <p class="text-muted">Birmingham Leader</p>
         <ul class="list-inline social-buttons">
           <li class="list-inline-item">
-            <a href="https://www.linkedin.com/in/lisabamberg/" target="_blank" class="shadow-sm">
+            <a href="https://www.linkedin.com/in/lisabamberg/" target="_blank" class="shadow-sm" aria-label="Navigation link to Lisa Bamberg LinkedIn Page">
```

Comment:
Screen readers will announce this as a link, so a better aria-label would be "Lisa Bamberg's LinkedIn Page"